icevision/models/fastai/unet/prediction.py

```diff
@@ -39,10 +39,21 @@ def convert_raw_predictions(
     keep_images: bool = False,
 ) -> List[Prediction]:
     mask_preds = raw_preds.argmax(dim=1)
-    tensor_images, *_ = batch
+
+    if len(batch) > 1:
+        tensor_images, tensor_gts = batch
+    else:
```

Comment:
Does this happen at inference?
E.g. when there are no masks?

Reply:
At inference time without GT masks being available, the second condition is triggered (and `tensor_gts` is set as an array of `None`).

This is admittedly a bit hacky, but was in still in an effort to get `unet` working in the more general semantic segmentation framework.